scripts/ci/verify_linter.py

```diff
@@ -70,9 +70,13 @@ def _cmd(cmd):
         check_call(cmd, shell=True)
 
     def add_from_code(self):
+        if self.extension_name == 'containerapp-preview':
+            self._cmd('azdev extension add containerapp')
         self._cmd('azdev extension add {}'.format(self.extension_name))
 
     def remove(self):
+        if self.extension_name == 'containerapp-preview':
+            self._cmd('azdev extension remove containerapp')
         self._cmd('azdev extension remove {}'.format(self.extension_name))
```

Comment:
Could we add some comments to explain this special logic, so as to make the reason why the logic does here easier to understand.

Reply:
Hi @wangzelin007 Could you help to handle this comment?